src/applications/find-va-forms/actions/index.js

```diff
@@ -17,11 +21,19 @@ export const fetchFormsFailure = () => ({
   type: FETCH_FORMS_FAILURE,
 });
 
-export const fetchFormsSuccess = response => ({
-  response,
+export const fetchFormsSuccess = results => ({
```

Comment:
Renamed this to `results` instead of `response`, since this will actually be the post-normalized response (e.g. the normalized list of forms).